src/config/server/server.js

```diff
@@ -1,31 +1,35 @@
-const express = require("express");
-const http = require("http");
-const cors = require("cors");
-const apiRouter = require("../../api/api_router");
-const config = require("../config.json");
-
-const app = express();
-const server = http.createServer(app);
-
-app.use(cors());
-app.use(express.json());
-app.use('/api', apiRouter);
-
-
-/**
- * Heartbeat endpoint
- */
-app.get('/', (req, res) => {
-  res.send('WebNexus API is running');
-});
-
-app.listen(config.httpPort, () => {
-  console.log(`Server is listening on :${config.httpPort}`);
-});
-
-server.listen(config.socketPort, () => {
-  console.log(`Socket Server is running on :${config.socketPort}`);
-});
-
-
-module.exports = { app, server };
+const express = require("express");
+const http = require("http");
+const cors = require("cors");
+const cookieParser = require('cookie-parser');
+
+const apiRouter = require("../../api/api_router");
+const config = require("../config.json");
+const checkAuthentication = require('../../api/middlewares/checkAuthentication');
+
+const app = express();
+const server = http.createServer(app);
+
+app.use(cors());
+app.use(cookieParser());
+app.use(express.json());
+app.use('/api', apiRouter);
+// app.use(checkAuthentication);

```

Comment:
Shouldn't the checkAuthentication be here? Wouldn't it be better to run it this way for all endpoints, although it would not be nice because it should not run for the register and login endpoints.

Reply:
That's why i did not put it here. Because it will be applied for the login and register.